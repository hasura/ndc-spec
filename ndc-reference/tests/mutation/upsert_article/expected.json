{
  "operation_results": [
    {
      "affected_rows": 1,
      "returning": [
        {
          "__value": null
        }
      ]
    }
  ]
}