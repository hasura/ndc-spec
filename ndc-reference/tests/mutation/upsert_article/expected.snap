---
source: ndc-reference/bin/reference/main.rs
expression: response.0
input_file: ndc-reference/tests/mutation/upsert_article/request.json
---
{
  "operation_results": [
    {
      "type": "procedure",
      "result": null
    }
  ]
}
